python/src/scipp/table_html/formatting_html.py

```diff
@@ -329,6 +331,7 @@ def summarize_variable(name,
     html += [
         f"<div class='xr-var-dtype'>{escape(repr(var.dtype))}</div>",
         f"<div class='xr-var-unit'>{escape(unit)}</div>",
+        f"<div class='xr-var-size'>{escape(size)} B</div>",
```

Comment:
Do we want to inclde this so prominently by default? It takes valuable space. Should it just be a tooltip? Or show up only as part of in the folded values view?

Reply:
Good point, I'll have a look into showing it is a less space hungry manner.

Reply:
To me, i would say it's ok to have it just in the title line, next to `scipp.DataArray` (and maybe put it inside brackets instead of writing `size xxxB`.

I don't think you need to have it on each coordinate or data line.
If you want to see the size of that particular item just view the repr for that item.
If you relaly want to have it there for each element, then I would agree that tooltip would be the way to go.

In addition, i really think it should format the output in a human readable format, i.e. kB, MB, GB (save space and much more user friendly)

Reply:
> To me, i would say it's ok to have it just in the title line, next to scipp.DataArray (and maybe put it inside brackets instead of writing size xxxB.

I agree, having it just in the title line sounds like a good suggestion!